docsource/modules130-140.rst

```diff
@@ -236,7 +236,7 @@ Module coverage 13.0 -> 14.0
 +--------------------------------------------+-------------------------------------------------+
 |l10n_be_edi                                 |                                                 |
 +--------------------------------------------+-------------------------------------------------+
-|l10n_be_invoice_bba                         |                                                 |
+|l10n_be_invoice_bba                         | Merged into l10n_be                             |
```

Comment:
Strictly speaking, the module should be considered merged for our purposes only in 15.0, as the module is still present in the Odoo 14 distribution and installable.

Reply:
Fixed.